### ISSUE TITLE
Board Request: Input on employer restrictions of TSC Director

### ISSUE THREAD

**Issue by mikeal** [nodejs/TSC#231]
In our last board meeting an issue was raising about a portion of the bylaws were not previously aware of:

```
Only one Member which is part of a group of Related Companies shall be entitled at one time to
have a representative on the Board or vote on matters voted on by the Members provided, 
however, that the (i) TSC Director or the Individual Director, but not both, may be an employee of 
a Platinum Member or one of its Related Companies, notwithstanding the fact that such Platinum 
Member is also represented on the Board by virtue of its right to appoint a Director under 
Sections 2.3(a) and 4.3(a) of these Bylaws, and (ii) the Individual Director may be an employee 
of a Gold or Silver Member or one of its Related Companies, notwithstanding the fact that 
such Member may already be represented on the Board by an employee elected to the Board 
of Directors.
```

The board asked for input from both the TSC and the broader community on this issue. Broader input will happen in threads in the `board` repo, this thread is specifically for the TSC to give their feedback.

As it stands, the TSC Director is employed by a company that also represents the Gold Membership Tier, making one of them not "in good standing."

The board would like to know what the TSC's preferred resolution to this issue is.

**Comment by nebrius** [comment]
Tagging with tsc-agenda so we can discuss during next weeks call. I think a good goal would be to formalize/vote-on-if-need-be a recommendation to the board during the meeting. That said, I'd also like that formalization to be just that, a formalization of what's discussed in this thread here.

I also want to remind everyone to keep this thread civil and remember that we have a [Code of Conduct](https://github.com/nodejs/node/blob/master/CODE_OF_CONDUCT.md) that covers the conversations here. There are differing opinions on the best approach here (which is good!), and sometimes those differences can become personal (which is not so good). Let's try and respect each other and try to remember that we all want what's best for the Foundation.

**Comment by nebrius** [comment]
For my part, after listening to the previous discussions, I think I lean towards separating this into two separate actions: 1) unwinding the current situation so that we're in compliance with the existing by-laws and 2) revisiting the current by-laws and make adjustments as necessary.

I like this approach because it removes the pressure to update the by-laws quickly, and removes the possibility that some people may view whatever action we take as "doling out favors for friends/insiders." (I would be really surprised if that would actually happen, but public appearance is important, unfortunately).

**Comment by jasnell** [comment]
Given the discussion, I think that's the right approach to propose to the board. I want to make sure, however, that both actions are taken. The bylaws need to be updated to allow the TSC Director to serve independently of their employer (regardless of corporate membership level), just as the Individual Member Directors do. 

If we have to unwind the stack to get there, then so be it. 

That said, the decision of which board member steps back to correct the current imbalance should be left up to the two members in question.

**Comment by mikeal** [comment]
Here's the thing, the board chose specifically *not* to take action "unwinding" this until the TSC gave input on what solution they'd like no, and on a longer term basis.

If the TSC prefers that the bylaws remain unchanged and that one of them (based on prior comments that would be Rod) steps down then that is what we'll bring to the next meeting.

**Comment by jasnell** [comment]
To be clear, my personal *preference* is that (a) the current board members complete their current terms and (b) the bylaws be altered as suggested. It's just not entirely clear if (a) is possible or appropriate given the situation of one of the two being "not in good standing".

**Comment by mikeal** [comment]
> To be clear, my personal preference is that (a) the current board members complete their current terms and (b) the bylaws be altered as suggested. It's just not entirely clear if (a) is possible or appropriate given the situation of one of the two being "not in good standing".

So, we had a quick rundown of what the impact of a board director not being in good standing was that I'll note here:

* We had to look back through prior votes to make sure that we would make quorum even without one of those directors in attendance. If we had had any split votes we'd also have to take that in to account.
* Going forward, we need to keep in mind that one of these directors does not count for quorum.

That's pretty much it. This doesn't mean the board does not want to resolve this as soon as it can, but you can see why there wasn't a sense of immediate urgency and that the board felt comfortable waiting for feedback from the TSC and the community about how to best resolve the problem.

There's almost 4 weeks until the next board meeting, that would seem to be plenty of time for the TSC to voice its preference for how to move forward and for our counsel to draft a bylaw amendment if you find it necessary.

My personal sense of how the board wanted to handle this was *not* to have a resolution simply extending good standing to the directors until the term is over. There seemed to be a preference for finding out what the rules should be long term and either amending or enforcing them. In other words, the purpose of getting feedback was so that we could solve the long term problem and use it to resolve the immediate issue and not to push forward a temporary solution to the immediate issue.

**Comment by hackygolucky** [comment]
tl;dr **Questions we need you to consider**
1. **What do you think about multiple positions on the Board being represented by a single company**, including the TSC chair and Individual Directors?
2. **How would you propose the above bylaws be changed** to address your position?

**Comment by jasnell** [comment]
Given that there has been nothing new on this issue, I'm pulling the `tsc-agenda` label off.

**Comment by MylesBorins** [comment]
Is this something that still needs to be discussed with the board?

**Comment by jasnell** [comment]
Not unless the board requires further input. I believe this issue has been resolved.

**Comment by williamkapke** [comment]
The TSC (at that time) wanted the restriction lifted- what was the Board's decision?

With Myles as the TSC director, it isn't a far reach to think that Google may want to re-run for a Silver seat. They are also disincentivize to move to Gold/Platinum.

**Comment by jasnell** [comment]
They would have no issue with moving to Platinum.
